package.json

```diff
@@ -121,13 +122,16 @@
 		"lodash": "4.17.21",
 		"minimatch": "3.0.5",
 		"minimist": "1.2.6",
+		"next": "13.4.12",
 		"node-fetch": "2.6.7",
 		"node-forge": "1.3.0",
 		"normalize-url": "4.5.1",
 		"nth-check": "2.0.1",
 		"object-path": "0.11.8",
 		"prismjs": "1.27.0",
 		"puppeteer": "19.8.5",
+		"react": "18.2.0",
```

Comment:
I was getting horrible React version mismatch errors. This standardizes the versions.